platform_umbrella/apps/common_core/lib/common_core/postgres/cluster.ex

```diff
@@ -121,6 +126,16 @@ defmodule CommonCore.Postgres.Cluster do
     end)
   end
 
+  defp maybe_put_backup_config(changeset) do
+    case get_field(changeset, :backup_config) do
+      nil ->
+        put_embed(changeset, :backup_config, %BackupConfig{type: :none})
```

Comment:
I don't think we want this; it will mean anytime there's a changeset on an existing pg cluster we always have to pass in the current backup config. Meaning there will be no way to do a partial edit.

It's probably better to add this onto the reading side.

Reply:
Yeah, we may not even need to. If we keep the semantic that nil is equivalent to none, right?